src/plus/integrations/providers/github/github.ts

```diff
@@ -365,6 +366,7 @@ export class GitHubApi implements Disposable {
 
 			return {
 				provider: provider,
+				id: author?.user?.login ?? '',
```

Comment:
I thought about this case. To me it like when we inspect a commit, see an author name + email there and cannot find any corresponding account on GitHub so we don't have a login=user-id. But we still want to represent author info somehow, therefore we don't want to cut the whole object nor throw an exception here. (Also we can see an author-name or just an email, or even nothing of that, but still want to pass through the converter what we have.

Now I'm thinking about letting id have `undefined` value, but I'm thinking about specifying of certain types of user objects where it's required. For example, a commit author easily can have 'undefined' id, because it can be a commit of a user who has never existed on GitHub, but a PR author or a reviewer, cannot be just an abstract name, it can only be a user of the service so they should have an id/login, so here we have rights to require it.

I'm going to review the changes and look what I can do with it.